### ISSUE TITLE
WSAFacebookApi_GetSwapChainPanel issue in UWP Il2cpp build

### ISSUE THREAD

**Issue by BasitFarooq17** [ClaytonIndustries/WSANative#4]
![Capture](https://user-images.githubusercontent.com/55625265/109797645-89b22400-7c3b-11eb-962e-b3af0d4e2eca.PNG)

It says that Api call has unresolved external symbol. Build fails. Kindly help me to fix this issue for Il2cpp build type. Thanks

**Comment by ClaytonIndustries** [comment]
Presently there's a bug in one of the source files, here's how to fix it

Edit WSAFacebookApi.cs and remove the following.

```
#if (ENABLE_IL2CPP && UNITY_WSA_10_0)
    [DllImport("__Internal")]
    private static extern Windows.UI.Xaml.Controls.SwapChainPanel GetSwapChainPanel();
#endif
```

**Comment by ClaytonIndustries** [comment]
This has been fixed in version 2.0 of the plugin
